src/theme/index.ts

```diff
@@ -23,14 +25,18 @@ const createPalette = (options: PaletteOptions) => {
     },
     secondary: {
       main: '#ff5a5f',
+      dark: midGrey,
+    },
+    warning: {
+      main: '#F2994A',
```

Comment:
This is the orange color for when the label is `Medium`.

Reply:
I'm cool with this. But I want to double check with Patrick that he envisions this color for all warning like usages.

Reply:
Ok sure! Happy to check in with Patrick on this.